examples/cicd-argocd/argocd/show-argocd-logs

```diff
@@ -11,5 +11,5 @@ source ${STARTUP_DIR}/../settings.sh
 KUBECTL="kubectl --context=${KUBE_CONTEXT}"
 
 NAMESPACE_NAME=argocd
-${KUBECTL} logs -f $(kubectl --context="${KUBE_CONTEXT}" get pods -n ${NAMESPACE_NAME} -l app=argocd-server -o jsonpath='{.items[0].metadata.name}') -n ${NAMESPACE_NAME}
+${KUBECTL} logs -f $(kubectl --context="${KUBE_CONTEXT}" get pods -n ${NAMESPACE_NAME} -l app.kubernetes.io/name=argocd-server -o jsonpath='{.items[0].metadata.name}') -n ${NAMESPACE_NAME}
```

Comment:
app.kubernetes.io/name is the new standard label format so argocd have switched to it